lib/charts/chart.rb

```diff
@@ -17,20 +17,34 @@ def validate_arguments(data, options)
     raise ArgumentError.new('Data missing') if data.empty?
     raise ArgumentError.new('Data not an array') unless data.is_a? Array
     raise ArgumentError.new('Options missing') unless options.is_a? Hash
-    if options[:outer_margin] and !options[:outer_margin].is_a?(Numeric)
-      raise ArgumentError.new('outer_margin not a number')
+    if options[:outer_margin] && !options[:outer_margin].is_a?(Numeric)
+      raise ArgumentError, 'outer_margin not a number'
     end
-    validate_array_and_count(data, options, :colors)
-    validate_array_and_count(data, options, :labels)
+    is_an_array?(options, [:labels, :colors])
```

Comment:
`validate_array_and_count` used to be the old title for a method which did two things 1) check if input is an array and: 2) validate the count. I splitted it in two separate methods and needed to find new names for them. I would pick `array?` if you can't think of a better one...